docs/pages/unified/engagement/engagement.yaml

```diff
@@ -477,7 +477,20 @@ paths:
                               type: string
                           required: equals
                       required: address
-                  required: emails
+                    tagIds:
+                      type: object
+                      properties:
+                        in:
```

Comment:
nit: according to the zod validators, `in` is required if `tagIds` is present. Same for `sequenceIds`

Reply:
oh yeah good catch, let me add